public/pages/CreateMonitor/containers/DefineMonitor/utils/mappings.js

```diff
@@ -43,6 +43,8 @@ export function getFieldsFromProperties(properties, dataTypes, path) {
 }
 
 export function getTypeFromMappings(mappings, dataTypes, path = '') {
```

Comment:
Do we want to remove these 2 comments?

Reply:
Want to keep here for people to understand structure of index mapping easily